### DISCUSSION TITLE
MriSenseNufft, AdjMriSenseNufft, AdjKbNufft, ToepSenseNufft removed

### DISCUSSION THREAD

**Discussion by kingaza** [mmuckley/torchkbnufft#61] [General]
Hello @mmuckley ,

Thank you very much for sharing such a great work.

When I try some open-source project, some errors happen, because some functions like MriSenseNufft, AdjMriSenseNufft, AdjKbNufft, ToepSenseNufft, etc., have been deprecated. 
If I still want to use the latest version of torchkbnufft, how can I solve this problem?

Thanks a lot.

Best Regards,
kingaza

**Comment by mmuckley** [comment]
Hello @kingaza, you can check out the documentation on [readthedocs](https://torchkbnufft.readthedocs.io/en/stable/generated/torchkbnufft.KbNufft.html). Equivalent functions are still present - they've just been renamed to make their purpose more clear since 1.0.

**Reply by kingaza** [reply]
thx for your reply.
